examples/docker_example/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/tarekbadrshalaan/GoKafka
```

Comment:
This should be `github.com/confluentinc/confluent-kafka-go/examples/docker_example`

Reply:
thanks for your comment @edenhill , the pull request updated now.